recipe-server/client/control_new/state/app/extensions/reducers.js

```diff
@@ -24,11 +24,22 @@ function items(state = new Map(), action) {
 
 function listing(state = new Map(), action) {
   switch (action.type) {
-    case EXTENSION_PAGE_RECEIVE:
+    case EXTENSION_PAGE_RECEIVE: {
+      let totalExtensions = state.get('all', new Set());
+
+      action.extensions.results.forEach(result => {
+        totalExtensions = totalExtensions.add(fromJS(result));
+      });
+
+      totalExtensions = totalExtensions.sort((a, b) => a.get('id') <= b.get('id'));
```

Comment:
This sort method won't work. Immutable's `.sort()` method (and JS's as well), want a comparator that returns `0`, `1`, or `-1`.